Argo/Types/StandardTypes.swift

```diff
@@ -19,6 +38,17 @@ extension Int: Decodable {
 }
 
 extension Int64: Decodable {
+  /**
+    Decode `JSON` into `Decoded<Int64>`.
+
+    Succeeds if the value is a number that can be converted to an `Int64` or a
+    string that represents a large number, otherwise it returns a type
+    mismatch.
```

Comment:
JSON can only hold numbers of a specific size (I don't remember what that size is) so if it's larger than that size it needs to be represented as a string. I'm not sure how to communicate this here.